src/bfastts.cpp

```diff
@@ -12,11 +12,15 @@ using namespace Rcpp;
 IntegerVector bfast_cpp_closestfrom(const NumericVector& a, const NumericVector& b, const bool twosided) {
   int na = a.size();
   int nb = b.size();
-  Rcpp::IntegerVector out(na);
+  IntegerVector out(na);
+  
+  if (na == 0 || nb == 0) {
```

Comment:
Should these check for < 0? I know it's not likely to happen, but who knows...

Reply:
Unlike C++ vectors, the size function of Rcpp vectors really returns a *signed* integer, so I guess this is a good idea ;-)